src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/DecisionWithoutHearingListed.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.iahearingsapi.domain.handlers.servicedatahandlers;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.iahearingsapi.domain.entities.AsylumCaseFieldDefinition.DECISION_WITHOUT_HEARING_LISTED;
+import static uk.gov.hmcts.reform.iahearingsapi.domain.entities.ServiceDataFieldDefinition.CASE_REF;
+import static uk.gov.hmcts.reform.iahearingsapi.domain.entities.ccd.field.YesOrNo.NO;
+import static uk.gov.hmcts.reform.iahearingsapi.domain.entities.ccd.field.YesOrNo.YES;
+import static uk.gov.hmcts.reform.iahearingsapi.domain.entities.hmc.HearingChannel.ONPPRS;
+import static uk.gov.hmcts.reform.iahearingsapi.domain.handlers.servicedatahandlers.HandlerUtils.isHearingChannel;
+import static uk.gov.hmcts.reform.iahearingsapi.domain.handlers.servicedatahandlers.HandlerUtils.isHmcStatus;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+import uk.gov.hmcts.reform.iahearingsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iahearingsapi.domain.entities.ServiceData;
+import uk.gov.hmcts.reform.iahearingsapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iahearingsapi.domain.entities.ccd.callback.ServiceDataResponse;
+import uk.gov.hmcts.reform.iahearingsapi.domain.entities.hmc.HmcStatus;
+import uk.gov.hmcts.reform.iahearingsapi.domain.handlers.ServiceDataHandler;
+import uk.gov.hmcts.reform.iahearingsapi.domain.service.CoreCaseDataService;
+
+@Component
+@RequiredArgsConstructor
+public class DecisionWithoutHearingListed implements ServiceDataHandler<ServiceData> {
+
+    public static final String CASE_TYPE_ASYLUM = "Asylum";
+
+    private final CoreCaseDataService coreCaseDataService;
+
+    @Override
+    public boolean canHandle(ServiceData serviceData) {
+        requireNonNull(serviceData, "serviceData must not be null");
+
+        return isHearingChannel(serviceData, ONPPRS)
+               && (isHmcStatus(serviceData, HmcStatus.LISTED)
```

Comment:
don't we need to check for hearing type? substantive  hearing. Also what is expected on back of updates, do we need to ignore updates ? then it has to be handled

Reply:
Not according to the bug ticket and the original ticket. I can check with Simon

Reply:
The business hasn't required to restrict this handling to substantive hearings only, even though they'll most probably be substantive. Simon will mention it to Pav.
Regarding updates, nothing gets triggered with updates, but also there's nothing being triggered with this on-papers listing either. So basically as long as the message contains "on papers" and "listed" it'll set a flag field to `Yes`, and to `No` if it's cancelled.

Reply:
I've added the restriction to for substantive hearings anyway because they'll be only substantive regardless, and it'll certainly exclude the possibility of bail cases somehow trying to trigger an event for asylum cases